ReadingList/ReadingListRecordResponse.swift

```diff
@@ -17,12 +17,6 @@ class ReadingListRecordResponse: ReadingListResponse {
     }
 
     var record: ReadingListServerRecord? {
-        get {
-            if let json = self.json {
-                return ReadingListServerRecord(json: json)
-            } else {
-                return nil
-            }
-        }
+        return json.flatMap { ReadingListServerRecord(json: $0) }
```

Comment:
This sort of re-write, while welcome, should probably be in its own PR.

Reply:
It probably should but if you could make an exception this one time, I would be really grateful. I've had this code for the last months and I would just kind of want to merge it in so that I can proceed with the rest.